package.json

```diff
@@ -11,9 +11,9 @@
   "scripts": {
     "start": "start-storybook -p 6006",
     "watch": "babel src -w -d dist --ignore src/stories",
```

Comment:
Does this `watch` not also need updating?

Reply:
This watch can be removed entirely, as development happens using storybook.